### ISSUE TITLE
Uninstall options not available for owned collection

### ISSUE THREAD

**Issue by Sonsalt** [SIXNetworks/withSIX-Platform#137]
I am not able anymore to "remove" my collection content, just unsubscribe, even though I am the owner. 

- [ ] Uninstall option for installed collections missing
- [ ]  Remove unsubscribe ability from owned collections

**Comment by sickboy** [comment]
@Sonsalt you wanted a single option (Remove or Unsubscribe), which then asks you if you also want to remove the content.
The only bug here, as it seems, is that Unsubscribe should be Remove, when you own the collection.

Please make sure you don't mix between various website endpoints though (staging/preview/production) or this might explain the desynchronization aswell.

**Comment by Sonsalt** [comment]
yes that is the only bug, it should say delete instead of unsubscribe

**Comment by sickboy** [comment]
@Sonsalt 

> Please make sure you don't mix between various website endpoints though (staging/preview/production) or this might explain the desynchronization aswell.

Confirm this please.

**Comment by Sonsalt** [comment]
@sickboy I only have one website open an in only one tab, just the staging nothing else. Not using Sync UI atm

**Comment by sickboy** [comment]
@Sonsalt the report originated from the same time you were reporting the login issues, so try a clean start and confirm, then we can move on :)

**Comment by sickboy** [comment]
@Sonsalt the title and description were out of date. Fixed.

**Comment by sickboy** [comment]
@Sonsalt this should be resolved on the next client update + following API sync (e.g when trying to perform an action with said collection)
